pam/integration-tests/cli_test.go

```diff
@@ -269,7 +269,7 @@ func saveArtifactsForDebug(t *testing.T, artifacts []string) {
 	}
 
 	// We need to copy the artifacts to a temporary directory, since the test directory will be cleaned up.
-	tmpDir := filepath.Join("/tmp/authd-test-artifacts", testutils.GoldenPath(t))
+	tmpDir := filepath.Join(os.TempDir(), "authd-test-artifacts", testutils.GoldenPath(t))
```

Comment:
Actually... Would you mind controlling this output path with an environment variable? It would be better for us to update it should the need come.

Reply:
You mean in our qa right? I was thinking the same, so explicitly setting `GOTMPDIR` in our tests...

Reply:
Yes and no. We should not touch the `GOTMPDIR` value. What I meant is:
```suggestion
        // If the env is not set, we do not save the artifacts.
        artifactPath := os.Getenv("AUTHD_TEST_ARTIFACTS_PATH")
	tmpDir := filepath.Join(artifactPath, testutils.GoldenPath(t))
```
Then, we set this environment in our QA action (and locally, should we want to), before running `go test`.

Reply:
Yeah, that was the other option in mind, but I'd still fallback to `TmpDir` for AUTHD_TEST_ARTIFACTS_PATH.

Reply:
Both are fine, TBH. Avoiding saving anything if we don't set the environment is better since it gives us more options to control the behaviour, so I'd go with this approach. (I updated the suggestion to give a better impression on the behavior)

Reply:
Any opinion on this, @didrocks @GabrielNagy?

Reply:
Fine by me too.